src/index.ts

```diff
@@ -690,6 +692,35 @@ class Envoy {
     this.http_server.listen(WH_SERVER_PORT);
   }
 
+  startStoragePolling () {
+    setInterval(this.updateStorageUsage.bind(this), 60 * 60 * 1000)
+  }
+
+  updateStorageUsage () {
+    const hashes = []
+    const usagePerDna = getDiskUsagePerDna(hashes)
+
+    // this fires off a bunch of promises and never waits for them to return
+    hashes.forEach(hash => {
+      const cellId = this.getServiceLoggerCellId(hash)
+
+      const payload = {
+        source_chains: [],
+        integrated_entries: [],
+        total_disk_usage: usagePerDna[hash]
```

Comment:
Right now this just logs the simplest thing, total disk space usage per dna. There's room in service logger to track much more detail disk usage, but we're not currently using it.